docs/release/release-team-onboarding.md

```diff
@@ -0,0 +1,61 @@
+are delivered smoothly and in a timely manner. While there are specific responsibilities for each sub-team within
+the Release Team Shadows, below are some general notes that every member of the Releease Team might benefit by going 
+through at the beginning of the cycle:
+
+- Slack:
+    - Most discussions related CAPI Release topics happens in the #cluster-api channel on the Kubernetes Slack. If you need access to the Kubernetes Slack, please visit http://slack.k8s.io/.
+- Kubernetes SIG membership:
+    -  Try to become an official member of the Kubernetes SIG, if possible. More information on the membership and requirements can be found [here](https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/release/release-team.md#cluster-api-release-team-vs-kuberneteskubernetes-sig-membership).
+- Familiarize yourself with the Release Process:
+    - Review the release [tasks document](../release/release-tasks.md) which explains the responsibilities and tasks for each role within the release team.
+- Check the Release Timeline:
+    - Go through the [release timeline](../release/releases) of the release cycle you are involved in (i.e checkout `release-1.6.md` if you are part of the 1.6 cycle release team) to better understand the key milestones and deadlines.
+
+Now, let's dive into the specific onboarding notes for each sub-team below.
+
+### Release Lead Shadows Team
+
+- Explore CI Setup and Tools:
+    - Gain some insights into how CI is setup in the Cluster API and tools used. You can check out CI Team onboarding [notes](#ci-signal-lead-shadows-team) for more details.
+- Learn about Release Notes Generation:
+    - Get to know how the release notes generation tools works.
+- Review the onboarding notes for both the Comms and CI Lead Shadows teams:
+    - Check out the onboarding notes of [Comms](#comms-lead-shadows-team) and [CI Signal](#ci-signal-lead-shadows-team) teams.
+
+### Comms Lead Shadows Team
+
+- Understand Release Process: 
+    - Get to know how project's release process works.
+    - Walk through the [release note generation process](../release/release-tasks.md#create-pr-for-release-notes) and try to generate notes by yourself. This is the most important process the comms team is in charge of.
+    - Familiarize yourself with the release notes tool [code](https://github.com/kubernetes-sigs/cluster-api/tree/main/hack/tools/release). You'll probably need to update this code during the release cycle to cover new cases or add new features.
+- Documentation familiarity:
+    - Explore project's documentation and start learning how to update and maintain it.
+- Communication channels:
+    - Learn about project's primary communication channels, including Slack, mailing list and office hours. Try to understand how information is distgributed and how to use them effectively for announcements, updates and discussions related to the release cycle.
+
+### CI Signal Lead Shadows Team
```

Comment:
```suggestion
### CI Signal Lead Team
```

Reply:
Removed the "Lead" as well to align with your previous suggestion.